nixos/modules/services/web-servers/apache-httpd2/separate.nix

```diff
@@ -0,0 +1,48 @@
+{ lib }:
+with lib;
+rec {
+  runtimeDir = "/run/httpd";
+
+  configParser = config:
```

Comment:
It looks like `<RequireAny>` and `<RequireAll>` aren't supported yet. How would those look in this new configuration?

Reply:
A hack-ish way could be:
```nix
[
  {
    RequireAny."" = {
      stuffHere = 5;
    };
  }
]
```
apache should just ignore the extra space, but I'm not sure I haven't tested it